packages/cozy-stack-client/src/AppCollection.spec.js

```diff
@@ -87,6 +86,36 @@ describe(`AppCollection`, () => {
             'Invalid "sources" attribute passed in query, please use an array with at least one element.'
           )
         }
+      }
+
+      it('should throw an error if query.sources is not an array', async () => {
+        const query = { sources: {} }
+        expectAssertionGet(query)
```

Comment:
Mmh, this test will always pass since expectAssertionGet is not awaited. `expectAssertionGet` is not very clear.

```jsx
await expect(collection.get('io.cozy.apps/fakeid', query)).toRejectWith('Invalid "sources" attribute passed in query, please use an array with at least one element.')
```

should be sufficient